src/panels/lovelace/cards/hui-entity-button-card.ts

```diff
@@ -54,32 +56,34 @@ implements LovelaceCard {
       return html``;
     }
     const stateObj = this.hass!.states[this.config.entity];
-    
+
+    applyThemesOnElement(this, this.hass!.themes, this.config.theme);
+
     return html`
       ${this.renderStyle()}
       <ha-card @click="${this.handleClick}">
         ${
-          !stateObj 
-          ? html`<div class="not-found">Entity not available: ${this.config.entity}</div>` 
-          : html`
+      !stateObj
+        ? html`<div class="not-found">Entity not available: ${this.config.entity}</div>`
+        : html`
               <paper-button>
                 <div>
                   <ha-icon
                     data-domain="${computeStateDomain(stateObj)}"
                     data-state="${stateObj.state}"
                     .icon="${this.config.icon ? this.config.icon : stateIcon(stateObj)}"
-                    style="${styleMap({filter: this._computeBrightness(stateObj), color: this._computeColor(stateObj)})}"
+                    style="${styleMap({ filter: this._computeBrightness(stateObj), color: this._computeColor(stateObj) })}"
                   ></ha-icon>
                   <span>
-                    ${this.config.name 
-                      ? this.config.name 
-                      : computeStateName(stateObj)
-                    }
+                    ${this.config.name
+            ? this.config.name
```

Comment:
Why is this reformatted? This doesn't look improved

Reply:
I didn't catch that. Format on save seems to be doing this

Reply:
I didn't catch that. Format on save seems to be doing this

Reply:
What are you using for format on save, are you using prettier ?

Reply:
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode